app/views/quizzes/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<h1>All Quizzes</h1>
+
+<% if current_user.num_incomplete_quizzes > 0 %>
+  <% num = current_user.num_incomplete_quizzes %>
+  <div class="incomplete-alert">
+    You have <%= pluralize(num, "incomplete quiz")%>!
+  </div>
+<% end %>
```

Comment:
One minor gripe: Could you change the method `num_incomplete_quizzes` to `incomplete_quizzes_count`? I might also switch a couple of lines to keep things DRY:

```
<% num = current_user.incomplete_quizzes_count %>
<% if num > 0 %>
  # ...
```